### ISSUE TITLE
Allow keyboard access in popover

### ISSUE THREAD

**Issue by triplef** [indragiek/INPopoverController#4]
Does anyone know how to enable keyboard access (using the tab key to navigate elements) in the popover window? For some reason this is currently not possible.

I tried manually calling recalculateKeyViewLoop but that doesn't seem to be enough. Additionally setting the initial first responder for the window allows keyboard access, but then I’m running into an assertion failure in AppKit when tabbing past the last element in the window:

> **\* Assertion failure in + [NSView _findFirstKeyViewInDirection:forKeyLoopGroupingView:], /SourceCache/AppKit/AppKit-1138.23/AppKit.subproj/NSView.m:11046
> this method is supposed to only be invoked on top level items

Any ideas are welcome! :)
